src/js/variant-widget/eva-variant-widget.js

```diff
@@ -1132,8 +1131,14 @@ function getProteinSubstitutionScore(consequenceTypes,so_array,source) {
         for (var j = 0; j < consequenceTypes[i].soTerms.length; j++) {
             if (consequenceTypes[i].soTerms[j].soName == _.first(so_array)) {
                 _.each(_.keys(consequenceTypes[i].proteinSubstitutionScores), function (key) {
-                    score = _.findWhere(this, {source: source}).score
-                    score_array.push(_.findWhere(this, {source: source}).score)
+                    var source_scores = _.findWhere(this, {source: source})
+                    if (source_scores !== undefined) {
+                        if (source_scores.score === undefined) {
+                            console.error('Found ' + source + ' object with no score');
```

Comment:
Not having a score is not an error, it simply wasn't generated by VEP. We could use `console.info` instead if we really want to log this situation.